store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreProperties.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2017 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.federatedstore;
+
+import uk.gov.gchq.gaffer.store.StoreProperties;
+
+import static uk.gov.gchq.gaffer.federatedstore.FederatedStoreConstants.AUTHS;
+import static uk.gov.gchq.gaffer.federatedstore.FederatedStoreConstants.CUSTOM_PROPERTIES_AUTHS;
+import static uk.gov.gchq.gaffer.federatedstore.FederatedStoreConstants.FILE;
+import static uk.gov.gchq.gaffer.federatedstore.FederatedStoreConstants.GAFFER_FEDERATED_STORE;
+import static uk.gov.gchq.gaffer.federatedstore.FederatedStoreConstants.KEY_GRAPH_IDS;
+import static uk.gov.gchq.gaffer.federatedstore.FederatedStoreConstants.KEY_SKIP_FAILED_FEDERATED_STORE_EXECUTE;
+import static uk.gov.gchq.gaffer.federatedstore.FederatedStoreConstants.PROPERTIES;
+import static uk.gov.gchq.gaffer.federatedstore.FederatedStoreConstants.SCHEMA;
+
+public class FederatedStoreProperties extends StoreProperties {
```

Comment:
Have a quick search through the code and update any federated store.properties so that the properties class is set to  uk.gov.gchq.gaffer.federatedstore.FederatedStoreProperties.

Perhaps you could update the getters and setters to make them not static and base this class on the other Store Properties classes we have, such as AccumuloProperties.

Reply:
👍